### ISSUE TITLE
A11y_SQLAzureDataStudio_SQLServerBigDataCluster_ServiceSettings_Keyboard:There is no keyboard key to dismiss the tooltip.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9290]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudio;#A11yWCAG2.1;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS1.4.13;#Keyboard;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MA1.4.13

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment".
4. Select "SQL Server Big Data Cluster" card under deployment options.
5. Select the version & checkbox.
6. Navigate till Selet button & hit enter.
7. Select Configuration profile --> Update Azure settings --> Update Cluster settings & hit Next.
8. Navigate to the information icon under "Storage settings" button.

**Actual:**
On navigating using tab to the information button under "Storage settings" there is a tooltip in the display but it does not dismiss with any keyboard key like an escape. On escape, the window is closed.

**Expected:**
There should be a keyboard key available to dismiss the tooltip.

**User Impact:**
Keyboard users will face difficulty to dismiss the tooltip without moving the focus from it.

**Attachment for Reference :**
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/EXkiWaBjvZJHh-UyfWQgPp0BVXkC4rTStv1CaOZcvEmY6A?e=vJvwC4

**Comment by alanrenmsft** [comment]
if you move the focus away from the icon, the tooltip will be dismissed.

**Comment by fsteffi** [comment]
GithubTags:#A11y_SQlADS_Win32;#A11yMAS;

**Comment by sspatil123** [comment]
As per PO comment big data cluster is removed. Hence closing bug. GithubTags:#Closed;Version 1.75

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
